astronomer/providers/amazon/aws/sensors/s3.py

```diff
@@ -81,7 +81,14 @@ def __init__(
 
     def execute(self, context: Context) -> None:
         """Check for a keys in s3 and defers using the trigger"""
-        if not self.poke(context):
+        try:
+            poke = self.poke(context)
+        except Exception as e:
+            if self.soft_fail:
+                raise AirflowSkipException(f"{self.task_id} failed")
```

Comment:
```suggestion
                raise AirflowSkipException(str(e))
```
To keep this consistent with the output that we use in the triggerer